<?xml version="1.0" encoding="UTF-8"?><testsuite name="nosetests" tests="1" errors="1" failures="0" skip="0"><testcase classname="hcc_router.hcc652.Hcc652" name="test_get_set" time="0.696"><error type="exceptions.ValueError" message='No JSON object could be decoded&#10;-------------------- &gt;&gt; begin captured logging &lt;&lt; --------------------&#10;requests.packages.urllib3.connectionpool: INFO: Starting new HTTPS connection (1): authentication-stg.apixio.com&#10;requests.packages.urllib3.connectionpool: DEBUG: "POST /auths HTTP/1.1" 200 None&#10;requests.packages.urllib3.connectionpool: INFO: Starting new HTTP connection (1): authentication-stg.apixio.com&#10;requests.packages.urllib3.connectionpool: DEBUG: "POST /tokens HTTP/1.1" 201 None&#10;requests.packages.urllib3.connectionpool: INFO: Starting new HTTPS connection (1): hcc-opprouter-stg.apixio.com&#10;requests.packages.urllib3.connectionpool: DEBUG: "GET /hcc/grinderusr1416591626@apixio.net/next_work_item HTTP/1.1" 404 0&#10;--------------------- &gt;&gt; end captured logging &lt;&lt; ---------------------'><![CDATA[  File "/usr/local/lib/python2.7/unittest/case.py", line 327, in run
    testMethod()
  File "/mnt/hcc_test_automation/hcc_router/hcc652.py", line 237, in test_get_set
    o = self.getNext()
  File "/mnt/hcc_test_automation/hcc_router/hcc652.py", line 223, in getNext
    return response.status_code, response.json()
  File "/usr/local/lib/python2.7/site-packages/requests/models.py", line 799, in json
    return json.loads(self.text, **kwargs)
  File "/usr/local/lib/python2.7/json/__init__.py", line 326, in loads
    return _default_decoder.decode(s)
  File "/usr/local/lib/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/local/lib/python2.7/json/decoder.py", line 384, in raw_decode
    raise ValueError("No JSON object could be decoded")
'No JSON object could be decoded\n-------------------- >> begin captured logging << --------------------\nrequests.packages.urllib3.connectionpool: INFO: Starting new HTTPS connection (1): authentication-stg.apixio.com\nrequests.packages.urllib3.connectionpool: DEBUG: "POST /auths HTTP/1.1" 200 None\nrequests.packages.urllib3.connectionpool: INFO: Starting new HTTP connection (1): authentication-stg.apixio.com\nrequests.packages.urllib3.connectionpool: DEBUG: "POST /tokens HTTP/1.1" 201 None\nrequests.packages.urllib3.connectionpool: INFO: Starting new HTTPS connection (1): hcc-opprouter-stg.apixio.com\nrequests.packages.urllib3.connectionpool: DEBUG: "GET /hcc/grinderusr1416591626@apixio.net/next_work_item HTTP/1.1" 404 0\n--------------------- >> end captured logging << ---------------------']]></error></testcase></testsuite>